Flow.Launcher.Plugin/Interfaces/IPublicAPI.cs

```diff
@@ -270,5 +270,11 @@ public interface IPublicAPI
         /// Non-C# plugins should use this method
         /// </summary>
         public void OpenAppUri(string appUri);
+
+        /// <summary>
+        /// Get Settings.LastQueryMode
+        /// Possible return values: "Empty", "Selected", "Preserved"
+        /// </summary>
+        public string GetLastQueryMode();
```

Comment:
why not use enum?

Reply:
Does enum support JSON rpc plugins? I wasn't sure.